docker/alpine/Dockerfile

```diff
@@ -31,8 +31,8 @@ RUN install_packages avahi bash ca-certificates docker jq openssh
 
 COPY docker/docker-init.sh init.sh
 
-RUN CLI_CMDS=$(jq -r '.commands | keys | map(.[0:index(":")]) | unique | join("\\ ")' < oclif.manifest.json); \
-    sed -ie "s/CLI_CMDS=\"help\"/CLI_CMDS=\"help\\ ${CLI_CMDS}\"/" init.sh && \
+# write CLI commands to a file to reference on init
```

Comment:
While this sed command works, it's nice to have the list of CLI commands in a separate file so we don't have to be concerned with breaking the sed matching when updating the init script in the future. Just a preference, your method worked well too!

Reply:
I had also considered it, but it means the extra file loading and grep'ing at runtime, versus sed at build time. I had opted to shift the cost to build time. It's very minor though, either way is fine.

Reply:
Good call, my change was missing the `help` command anyway so I just reverted to what you added.